Code/client/Games/Skyrim/Forms/LoadedCellData.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <Components/BaseFormComponent.h>
+
+struct LoadedCellData
+{
+    uint8_t pad160[0x160]; // Offset
```

Comment:
For padding naming, we usually do "pad" and then the current offset, so that'd be "pad0" here.